packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.css

```diff
@@ -0,0 +1,19 @@
+.pre_class {
+    width: 700px;
+    height: 175px;
+    top: 20px;
+    bottom: 20px;
+    text-align: left;
+    background: #FFFFFF;
+    color: #706E6B;
+    font-family: Arial;
```

Comment:
Why are you forcing the `Arial` font on a `pre` element?

Reply:
I follow the mockup